defaults/main.yml

```diff
@@ -14,4 +14,4 @@ alertmanager_irc_relay_irc_nickname_password: ""
 alertmanager_irc_relay_irc_realname: ""
 
 alertmanager_irc_relay_notice_once_per_alert_group: "yes"
-alertmanager_irc_relay_notice_template: "{% raw %}Alert {{ .Labels.alertname }} on {{ .Labels.instance }} is {{ .Status }}{% endraw %}"
+alertmanager_irc_relay_notice_template: "{% raw %}{{ range .Alerts }}Alert {{ .Labels.alertname }} on {{ .Labels.instance }} is {{ .Status }}{{ end }}{% endraw %}"
```

Comment:
This results in an unquoted string starting from `{`, which causes problems in alertmanager-irc-relay as it tries to serialize into map and not string. Simple solution would be to edit config template and quote this variable. Following should work:

```patch
diff --git a/templates/alertmanager-irc-relay-config.yml.j2 b/templates/alertmanager-irc-relay-config.yml.j2
index ad4c00c..8ce5baa 100644
--- a/templates/alertmanager-irc-relay-config.yml.j2
+++ b/templates/alertmanager-irc-relay-config.yml.j2
@@ -34,7 +34,7 @@ notice_once_per_alert_group: {{ alertmanager_irc_relay_notice_once_per_alert_gro
 # Define how IRC messages should be formatted.
 #
 # The formatting is based on golang's text/template.
-notice_template: {{ alertmanager_irc_relay_notice_template }}
+notice_template: {{ alertmanager_irc_relay_notice_template | quote }}
 # Note: When sending only one notice per alert group the default
 # notice_template is set to
 # "Alert {% raw %}{{ .GroupLabels.alertname }} for {{ .GroupLabels.job }} is {{ .Status }}{% endraw %}"
```